src/pages/SudoPrompt.vue

```diff
@@ -53,13 +53,13 @@ type SudoReason = 'networking' | 'docker-socket';
 const SUDO_REASON_DESCRIPTION: Record<SudoReason, {title: string, description: string}> = {
   networking: {
     title:       'Configure networking',
-    description: `This is used to provide bridged networking so that it is easier to access your
-                  containers.  If this is not allowed, containers can only be accessed via port
-                  forwarding.`,
+    description: `Used to provide bridged networking so that it is easier to access your
```

Comment:
I think we can distill this down, similar to the docker socket description below. 

```suggestion
    description: `Provides bridged networking so that it is easier to access your
```